extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -18,21 +25,28 @@ describes.realWin(
     },
   },
   (env) => {
+    let win;
+    let storeService;
+    let userWarnStub;
+
+    beforeEach(async () => {
+      win = env.win;
+      storeService = getStoreService(win);
+      userWarnStub = env.sandbox.stub(user(), 'warn');
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+    });
+
     let pageElement;
 
-    const defaultInlineConfig = {
-      items: [
-        {
-          'productId': 'city-pop',
-          'productTitle': 'Plastic Love',
-          'productPrice': 19,
-          'productPriceCurrency': 'JPY',
-        },
-      ],
-    };
+    it('should build shopping config component', async () => {
+      await createAmpStoryShoppingConfig();
+      expect(() => getShoppingConfig(pageElement)).to.not.throw();
+    });
 
     const keyedDefaultInlineConfig = {
```

Comment:
Please move this variable initialization to be above the new test methods in this file